contents.css

```diff
@@ -133,3 +133,47 @@ a > img {
 	border: none;
 	outline: 1px solid #0782C1;
 }
+
+.wcm-slider__images {

```

Comment:
Kan je de styling verwijderen. Deze moet in de module staan en niet in de core.
Ik heb de styling reeds overgenomen.